http/css/style.css

```diff
@@ -20,12 +20,17 @@
 body {
     font-size: 85%;
     font-family: sans-serif;
-    background-color: #ffffff;
     display: flex;
     justify-content: center;
+    background: url("../images/background.png") left / cover no-repeat fixed;
```

Comment:
I think I won't integrate the background image. It does give character to the page, but it also make it less readable and since there are already lots of informations in the page I'd rather keep the rest of the page clean.